src/connector/src/source/google_pubsub/mod.rs

```diff
@@ -62,12 +69,17 @@ pub struct PubsubProperties {
     /// in pub/sub because they guarantee retention of:
     /// - All unacknowledged messages at the time of their creation.
     /// - All messages created after their creation.
-    /// Besides retention guarantees, timestamps are also more precise than timestamp-based seeks.
+    /// Besides retention guarantees, snapshots are also more precise than timestamp-based seeks.
     /// See [Seeking to a snapshot](https://cloud.google.com/pubsub/docs/replay-overview#seeking_to_a_timestamp) for
     /// more details.
     #[serde(rename = "pubsub.start_snapshot")]
     pub start_snapshot: Option<String>,
 
+    /// `parallelism` is the number of parallel consumers to run for the subscription.
+    /// TODO: use system parallelism if not set
+    #[serde(rename = "pubsub.parallelism")]
```

Comment:
```
#[serde(rename = "pubsub.parallelism")]
#[serde_as(as = "DisplayFromStr")]
pub parallelism: Option<u32>,
```

Reply:
IIRC `DisplayFromStr` doesn't work for `Option<T>`.

Reply:
Oh sorry, I mean `Option<DisplayFromStr>`